octree/include/pcl/octree/octree_search.h

```diff
@@ -339,8 +339,8 @@ class OctreePointCloudSearch
     prioPointQueueEntry() : point_idx_(0), point_distance_(0) {}
 
     /** \brief Constructor for initializing priority queue entry.
-     * \param[in] point_idx an index representing a point in the dataset given by \a
-     * setInputCloud \param[in] point_distance distance of query point to voxel center
+     * \param[in] point_idx index for a dataset point given by \a setInputCloud
+     * \param[in] point_distance distance of query point to voxel center
```

Comment:
Oops, my bad. This must slipped in through my reformat PR 😅

Reply:
Eh it's mostly clang-format's fault though, I tried to fix it and it pushed it back to where it was, so I had to shorten the sentence.